### ISSUE TITLE
master does not build.

### ISSUE THREAD

**Issue by pnrobinson** [exomiser/Exomiser#315]
This is what I get (on the master branch; I got similar results on the develop branch)
```
(... everything OK up to here...)
[INFO] -------------------------------------------------------------
[ERROR] COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
[ERROR] /home/robinp/GIT/Exomiser/exomiser-core/src/main/java/org/monarchinitiative/exomiser/core/prioritisers/util/OffHeapDataMatrix.java:[27,24] cannot find symbol
  symbol:   class PreDestroy
  location: package javax.annotation
[ERROR] /home/robinp/GIT/Exomiser/exomiser-core/src/main/java/org/monarchinitiative/exomiser/core/prioritisers/util/OffHeapDataMatrix.java:[69,6] cannot find symbol
  symbol:   class PreDestroy
  location: class org.monarchinitiative.exomiser.core.prioritisers.util.OffHeapDataMatrix
[INFO] 2 errors 
[INFO] -------------------------------------------------------------
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[INFO] 
[INFO] exomiser ........................................... SUCCESS [ 13.943 s]
[INFO] exomiser-core ...................................... FAILURE [  6.431 s]
[INFO] exomiser-spring-boot-autoconfigure ................. SKIPPED
[INFO] exomiser-spring-boot-starter ....................... SKIPPED
[INFO] exomiser-spring-boot-test .......................... SKIPPED
[INFO] exomiser-cli ....................................... SKIPPED
[INFO] exomiser-data-genome ............................... SKIPPED
[INFO] exomiser-data-phenotype ............................ SKIPPED
[INFO] exomiser-rest-prioritiser .......................... SKIPPED
[INFO] exomiser-web ....................................... SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 25.939 s
[INFO] Finished at: 2019-02-25T16:52:29-05:00
[INFO] Final Memory: 41M/176M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.7.0:compile (default-compile) on project exomiser-core: Compilation failure: Compilation failure: 
[ERROR] /home/robinp/GIT/Exomiser/exomiser-core/src/main/java/org/monarchinitiative/exomiser/core/prioritisers/util/OffHeapDataMatrix.java:[27,24] cannot find symbol
[ERROR]   symbol:   class PreDestroy
[ERROR]   location: package javax.annotation
[ERROR] /home/robinp/GIT/Exomiser/exomiser-core/src/main/java/org/monarchinitiative/exomiser/core/prioritisers/util/OffHeapDataMatrix.java:[69,6] cannot find symbol
[ERROR]   symbol:   class PreDestroy
[ERROR]   location: class org.monarchinitiative.exomiser.core.prioritisers.util.OffHeapDataMatrix
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <goals> -rf :exomiser-core
robinp@ldg-jgm004:~/GIT/Exomiser$ java -version
java version "1.8.0_181"
Java(TM) SE Runtime Environment (build 1.8.0_181-b13)
Java HotSpot(TM) 64-Bit Server VM (build 25.181-b13, mixed mode)
```

**Comment by pnrobinson** [comment]
Note that I have been experimenting with Java 11 and using Ubuntu's switch-java-alternatives script to go back and forth. I am definitely on Java 8 right now, but since Java 11 does not include javax.annotation (which is one of the two problems above), I wonder if that could have something to do with it. I am uncertain of how to debug this.

**Comment by julesjacobsen** [comment]
The javax.annotation jar is included in version 12.0.0. Did you run this using ```./mvnw clean install```?

**Comment by julesjacobsen** [comment]
@pnrobinson I'm closing this as master does build on the CI server. Builds for java version 11 should work on exomiser 12.0.0+

If you still have issues, please re-open this.
